Formula/x/xz.rb

```diff
@@ -1,11 +1,9 @@
 class Xz < Formula
   desc "General-purpose data compression with high compression ratio"
   homepage "https://tukaani.org/xz/"
-  # The archive.org mirror below needs to be manually created at `archive.org`.
   url "https://github.com/tukaani-project/xz/releases/download/v5.6.3/xz-5.6.3.tar.gz"
   mirror "https://downloads.sourceforge.net/project/lzmautils/xz-5.6.3.tar.gz"
-  mirror "https://archive.org/download/xz-5.6.3.tar/xz-5.6.3.tar.gz"
-  mirror "http://archive.org/download/xz-5.6.3.tar/xz-5.6.3.tar.gz"
+  mirror "http://downloads.sourceforge.net/project/lzmautils/xz-5.6.3.tar.gz"
```

Comment:
```suggestion
  mirror "https://downloads.sourceforge.net/project/lzmautils/xz-5.6.3.tar.gz"
  mirror "http://downloads.sourceforge.net/project/lzmautils/xz-5.6.3.tar.gz"
```
Think you need to do this to avoid the audit failure

Reply:
That's already there a line up. The audit needs fixed: https://github.com/Homebrew/brew/pull/18631